be/src/exec/tablet_sink.cpp

```diff
@@ -204,9 +204,7 @@ Status NodeChannel::open_wait() {
             }
         } else {
             _cancelled = true;
-            LOG(WARNING) << name() << " add batch req success but status isn't ok, " << print_load_info()
-                         << ", node=" << node_info()->host << ":" << node_info()->brpc_port
-                         << ", errmsg=" << status.get_error_msg();
+            _err_st = status;
```

Comment:
why remove load_info from log ?

Reply:
Because the error message is already printed in the lower code. This place is repeated.